executor/cvsTask/cvstask.go

```diff
@@ -31,24 +30,25 @@ type strPair struct {
 }
 
 type Config struct {
+	Idx      int    `json:"Idx"`
 	SrcHost  string `json:"SrcHost"`
-	SrcDir   string `json:"SrcDir"`
 	DstHost  string `json:"DstHost"`
-	DstDir   string `json:"DstDir"`
-	StartLoc int64  `json:"StartLoc"`
+	DstDir   string `json:"DstIdx"`
+	StartLoc string `json:"StartLoc"`
 }
```

Comment:
Should we update the demo task config in `sample/config/demo.json`

Reply:
Good catch. Let me think about how to launch demo job by master-client ...

Reply:
Finished, We can still use master-client to test locally, which need to add flags to demo-server, such as `./bin/demoserver -d /datax -a 0.0.0.0:1234 -r 1000000`  PTAL